src/nevermined/api/SearchApi.ts

```diff
@@ -123,4 +123,74 @@ export class SearchApi extends Instantiable {
     }
     return this.query(query)
   }
+
+  /**
+   *
+   * @param contractAddress - The address of the NFT-721 subscription contract
+   * @param offset - The number of results to return
+   * @param page
+   * @param sort - The sort order
+   * @param appId - The appId used to filter the results
+   *
+   * @returns {@link Promise<QueryResult>}
+   */
+  public async bySubscriptionContractAddress(
```

Comment:
Is there any possibility to create a query to get all the ddos which are from NFT subscriptions by passing the publisher address? I mean to get all the subscription published by a passed wallet address

Reply:
it is